Tests/SwiftLintFrameworkTests/ConfigurationTests.swift

```diff
@@ -449,4 +450,45 @@ extension ConfigurationTests {
         let filenames = paths.map { $0.bridge().lastPathComponent }.sorted()
         XCTAssertEqual(filenames, ["Level1.swift"])
     }
+
+    func testDictInitWithCachePath() throws {
+        let configuration = try Configuration(
+            dict: ["cache_path": "cache/path/1"]
+        )
+
+        XCTAssertEqual(configuration.cachePath, "cache/path/1")
+    }
+
+    func testDictInitWithCachePathFromCommandLine() throws {
+        let configuration = try Configuration(
+            dict: ["cache_path": "cache/path/1"],
+            cachePath: "cache/path/2"
+        )
+
+        XCTAssertEqual(configuration.cachePath, "cache/path/2")
+    }
+
+    func testMainInitWithCachePath() {
+        let configuration = Configuration(
+            configurationFiles: [],
+            cachePath: "cache/path/1"
+        )
+
+        XCTAssertEqual(configuration.cachePath, "cache/path/1")
+    }
+
+    func testMainInitWithCachePathAndCachedConfig() {
+        let configuration1 = Configuration(
+            configurationFiles: [],
+            cachePath: "cache/path/1"
+        )
+
+        let configuration2 = Configuration(
+            configurationFiles: [],
+            cachePath: "cache/path/2"
+        )
+
+        XCTAssertEqual(configuration1.cachePath, "cache/path/1")
+        XCTAssertEqual(configuration2.cachePath, "cache/path/1")
+    }
```

Comment:
I think this test demonstrates another bug: when the `Configuration` is obtained from the in-memory cache, the `cachePath` is not taken into account.

Could anybody confirm this is the case?

Reply:
Yeah should be fine in practice. Thanks for checking though.